packages/big-design/package.json

```diff
@@ -32,61 +32,61 @@
     "typings": "dist/index.d.ts"
   },
   "dependencies": {
-    "@babel/runtime": "^7.15.4",
+    "@babel/runtime": "^7.24.4",
     "@bigcommerce/big-design-icons": "workspace:^",
     "@bigcommerce/big-design-theme": "workspace:^",
     "@popperjs/core": "^2.11.6",
     "@types/react-datepicker": "^6.2.0",
     "date-fns": "3.6.0",
     "downshift": "9.0.4",
-    "focus-trap": "^7.0.0",
-    "polished": "^4.0.0",
+    "focus-trap": "^7.5.4",
+    "polished": "^4.3.1",
     "react-beautiful-dnd": "^13.1.1",
     "react-datepicker": "^6.6.0",
```

Comment:
I also couldn't bump this to the latest as they added an [import from `date-fns`](https://github.com/Hacker0x01/react-datepicker/compare/v6.6.0...v6.8.0#diff-c5ae56d754e9b921eb7696fed16bdde4b1548e7786fb0edb59ac711e6c378041R16) that causes the next.js build to break. I think we need to look at getting rid of the custom babel build in our docs site.